File: workspace/people/p02-site-reliability-engineer/home/services/checkout/prod/oncall/operations/commands/2026-07-13/source/2026-07-13-checkout-oncall-database-failover-automation.py
"""Northstar Transit Cloud; Checkout SVC-17 incident context; 2026-07-13; error budget 2.4%; INC-260713."""
SERVICE = "Checkout SVC-17"
INCIDENT = "INC-260713"
TOPIC = "database_failover"
SPINE_IDS = ['org.ntc', 'inc.inc-260713', 'rel.oncall.2026-07-13', 'date.2026-07-13', 'term.no-kio-claim']
FINGERPRINT = "d1ae4893c4ece080"

def assess_failover_evidence(status: str) -> bool:
    return status == 'reviewed'

File: workspace/people/p02-site-reliability-engineer/home/services/checkout/prod/oncall/operations/commands/2026-07-13/source/2026-07-13-checkout-oncall-database-failover-automation.rs
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
pub const TOPIC: &str = "database_failover";
pub const FINGERPRINT: &str = "653cceb2368ec731";
pub fn failover_evidence_reviewed(status: &str) -> bool { status == "reviewed" }
